addons/reloadlaunchers/initSettings.sqf

```diff
@@ -0,0 +1,8 @@
+[
+    QGVAR(displayStatusText),
+    "CHECKBOX",
+    [LSTRING(SettingDisplayStatusTextName), LSTRING(SettingDisplayStatusTextDesc)],
+    ELSTRING(common,ACEKeybindCategoryWeapons),
```

Comment:
Atm, this will create another category also named "ACE Weapons". If #9343 is merged, it should results in 1 category.